cmd/heartbeats/main.go

```diff
@@ -43,13 +50,37 @@ func init() {
 	flag.StringVar(&periodStr, "period", "5", "the number of seconds between heartbeats")
 }
 
+type envConfig struct {
+	// Sink URL where to send heartbeat cloudevents
+	Sink string `envconfig:"SINK"`
+
+	// Name of this pod.
+	Name string `envconfig:"POD_NAME"`
+
+	// Namespace this pod exists in.
+	Namespace string `envconfig:"POD_NAMESPACE"`
+
+	// The ip address of this pod.
```

Comment:
is this used anywhere?

Reply:
It is not, but I was so excited about finding that k8s could do this I put it in here so it can be known... I can take it out :(